VirtualLibrary.Utility/Interfaces/ISeedingService.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace VL.Utility.Interfaces
+{
+    public interface ISeedingService
+    {
+        Task SeedBooksAsync(int count);
```

Comment:
Add a method that seeds a user in the system with the provided role

Task SeedUserAsync(string username, Roles role, string defaultPassword = "Password123!")

Roles is an enum that may define the two users in the system, "Librarian" and "User"